api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -117,5 +117,30 @@
                                      baseTableName="patient_program_attribute" baseColumnNames="voided_by"
             referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
+
+	<changeSet id="20180214-1000" author="Kelechi+iheanyi">
+		<preConditions onFail="MARK_RAN">
+			<not><tableExists tableName="condition"/></not>
+		</preConditions>
+		<comment>
+			Add condition table
+		</comment>
+		<createTable tableName="condition">
+			<column name="condition_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="additional_detail" type="varchar(255)">
+				<constraints nullable="true"/>
+			</column>
+			<column name="condition" type="varchar(50)" defaultValue="null"/>
+			<column name="clinical_status" type="varchar(50)" >
```

Comment:
Did you include all the fields mentioned in the ticket description?

Reply:
I have included the previous version field